test/versioned-routes.test.js

```diff
@@ -569,3 +577,38 @@ test('Vary header check (for documentation example)', t => {
     t.strictEqual(res.headers.vary, undefined)
   })
 })
+
+test('Should trigger a warning when a versioned route is registered via version option', t => {
```

Comment:
Should there be a `t.tearDown` for this test to remove the proxying?

Reply:
There is no need, it does not change the global cache.

Reply:
👍 

I'm accustomed to `mock-require`.